### PR TITLE
refactor(signin): Use new verify account sync template and add location data to others

### PR DESCRIPTION
**Pull request by vbudhram**
This PR uses the new verify account sync template and adds location data to password change completed and password change requested. It requires 

- [x] Update shrinkwrap when https://github.com/mozilla/fxa-auth-mailer/pull/243 merges

Connects with https://github.com/mozilla/fxa-auth-mailer/issues/189

### CODE REVIEW

**Comment by vbudhram** [comment]
@vladikoff Auth server portion of https://github.com/mozilla/fxa-auth-mailer/issues/189, r?

File: test/remote/account_create_tests.js [hunk 1]
```diff
@@ -53,12 +53,12 @@ describe('remote account create', function() {
...
 
   it(
-    'create and verify account',
+    'create and verify sync account',
     () => {
       var email = server.uniqueEmail()
       var password = 'allyourbasearebelongtous'
       var client = null
-      return Client.create(config.publicUrl, email, password)
+      return Client.create(config.publicUrl, email, password, {service: 'sync'})
```
Comment (vladikoff) [inline test/remote/account_create_tests.js]: @vbudhram do we need a test for a non `service: 'sync'` email?

Comment (vbudhram) [inline test/remote/account_create_tests.js]: Looks like we had tests but didn't verify correct template, added checks!

**Comment by vbudhram** [comment]
@vladikoff Thanks!

**Pull request merged**